tests/test_interface_svi.rb

```diff
@@ -170,38 +121,31 @@ def test_svi_set_management_default
     svi.destroy
   end
 
-  def test_svi_get_svis
+  def test_get_svis
     count = 5
 
-    ref = cmd_ref_autostate
     # Have to account for interface Vlan1 why we add 1 to count
     (2..count + 1).each do |i|
       str = 'Vlan' + i.to_s
       svi = Interface.new(str)
-      assert_result(ref.test_config_result(false),
-                    'Error: svi autostate not set to false') do
-        svi.svi_autostate = false
-      end
+      svi.svi_autostate = false
+      refute(svi.svi_autostate, 'Error: svi autostate not set to false')
       svi.svi_management = true
     end
 
     svis = Interface.interfaces
-    ref = cmd_ref_autostate
-    result = ref.default_value
     svis.each do |id, svi|
       case id
       when /^vlan1$/
-        result = true if ref.config_set?
-        assert_equal(result, svi.svi_autostate,
+        assert_equal(svi.default_svi_autostate, svi.svi_autostate,
                      'Error: svis collection, Vlan1, incorrect autostate')
         refute(svi.svi_management,
                'Error: svis collection, Vlan1, incorrect management')
       when /^vlan/
-        result = false if ref.config_set?
-        assert_equal(result, svi.svi_autostate,
-                     "Error: svis collection, Vlan#{id}, incorrect autostate")
+        refute(svi.svi_autostate,
+               "Error: svis collection, #{id}, incorrect autostate")
```

Comment:
Seems like this should be `assert_equal(svi.default_svi_autotstate, svi.svi_autostate, ...)` as well